addons/project/models/res_partner.py

```diff
@@ -61,3 +61,27 @@ def _create_portal_users(self):
                 'active': True,
             })
         return created_users
+
+    def action_view_tasks(self):
+        self.ensure_one()
+        action = {
+            'type': 'ir.actions.act_window',
+            'res_model': 'project.task',
+            'name': _("%(name)s's Tasks", name=self.name),
```

Comment:
```suggestion
            'name': _("%(project_name)s's Tasks", project_name=self.name),
```
Slightly more precise context for translators (so that the [animacy](https://en.wikipedia.org/wiki/Animacy) of the tasks' owner is made clear)

Reply:
Good catch! I changed it as `partner_name` in this case as we are in the partner model :)

Reply:
oh whoops, it even confused me :see_no_evil: